docs/runtime-contract.md

```diff
@@ -35,15 +35,15 @@ The
 ([v1.0.1](https://github.com/opencontainers/runtime-spec/blob/v1.0.1/spec.md))
 is the basis for this document. When this document and the OCI specification
 conflict, this document is assumed to override the general OCI recommendations.
-Where this document does not specify behavior, developers should assume an OCI
-compliant underlying implementation. Additionally, the core Knative definition
+Where this document does not specify behavior, runtime implementations SHOULD
+be OCI compliant with respect to those features.  Additionally, the core Knative definition
 assumes the
 [Linux Container Configuration](https://github.com/opencontainers/runtime-spec/blob/master/config-linux.md).
 
 In particular, the default Knative implementation relies on Kubernetes behavior
 to implement container operation. In some cases, current Kubernetes behavior in
-2018 is not as performant as recommended in this documentation. The goal of the
-Knative authors is to push as much of the required functionality into Kubernetes
+2018 is not as performant as specified in this documentation. The goal of the
```

Comment:
I don't think this is "recommended", I think this is "envisioned".

Reply:
fixed